src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -647,6 +684,17 @@ grpc_chttp2_transport::grpc_chttp2_transport(
     }
   }
 
+  if (channel_args.GetBool(GRPC_ARG_SECURITY_FRAME_ALLOWED).value_or(false)) {
+    transport_framing_endpoint_extension = QueryExtension<
+        grpc_core::TransportFramingEndpointExtension>(
+        grpc_event_engine::experimental::grpc_get_wrapped_event_engine_endpoint(
+            ep.get()));
+    if (transport_framing_endpoint_extension != nullptr) {
+      transport_framing_endpoint_extension->SetSendFrameCallback(
+          absl::bind_front(&grpc_chttp2_transport::WriteSecurityFrame, this));
```

Comment:
Prefer lambda capture:

`[this](xyz) { WriteSecurityFrame(xyz); }`

Reply:
Done.